pkg/ebpfev_v1/event_interface.go

```diff
@@ -0,0 +1,14 @@
+package ebpfev_v1
```

Comment:
[Required]: `ebpfev_v1` -> `ebpfev/v1` and the package should be named  `ebpfev` (notice, `v1` is a directory nested in `ebpfev`)

Reply:
done